wordpress-mobile/WordPress-Android
# Use shared debug store

This PR updates the build configuration in order to make debug builds use a shared keystore instead of the default one. 
The keystore is also used by the new Jalapeno flavor.

To test:
_Wasabi debug build_
1. Checkout this branch.
2. Rename your local default debug keystore  (`~/.android/debug.keystore`) so that it can't be found by gradle. 
3. Build `WasabiDebug`, run it and verify that Google Login doesn't work. 
4. Run `./gradlew prodDeps`, verify that your configuration gets updated and that the shared keystore is copied to your `~/.android/` folder. 
5. Build `WasabiDebug`, run it and verify that Google Login now works.

_Installable builds_
6. Download the Jalapeno installable apk from this PR and verify that it can be installed and that Google Login works.

_Wrap up:_
 7. Restore your default keystore. 

**Note:** When everything is ready, I'll release a new version of the release-toolkit and update the reference before merging. 

PR submission checklist:
